### PR TITLE
*: release v0.18.0

### PR DESCRIPTION
**Pull request by estroz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,59 @@
+## v0.18.0
+
+### Additions
+
+- The Ansible operator now includes a healthz endpoint and liveness probe. All operators will now have a running healthz endpoint (not publicly exposed) without changes. Existing Oper. ([#2761](https://github.com/operator-framework/operator-sdk/pull/2761))
```
Comment (hasbro17) [inline CHANGELOG.md:5]: I imagine this is probably coming over from the CHANGELOG fragment.
```suggestion
- The Ansible operator now includes a healthz endpoint and liveness probe. All operators will now have a running healthz endpoint (not publicly exposed) without changes. ([#2761](https://github.com/operator-framework/operator-sdk/pull/2761))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,59 @@
...
+- Add 'run local' subcommand, which has the same functionality of the deprecated 'run --local' mode. ([#3067](https://github.com/operator-framework/operator-sdk/pull/3067))
+- Add scorecard-test image push targets into Makefile. ([#3107](https://github.com/operator-framework/operator-sdk/pull/3107))
+
+### Changes
+
+- In Helm-based operators, reconcile logic now uses three-way strategic merge patches for native kubernetes objects so that array patch strategies are correctly honored and applied. ([#2842](https://github.com/operator-framework/operator-sdk/pull/2842))
+- Revert deprecation of the package manifests format. See [#2755](https://github.com/operator-framework/operator-sdk/pull/2755) for deprecation details. The package manifests format is still officially supported by the Operator Framework. ([#2944](https://github.com/operator-framework/operator-sdk/pull/2944))
+- 'bundle validate' will print errors and warnings from validation. ([#3083](https://github.com/operator-framework/operator-sdk/pull/3083))
+- **Breaking change**: Set bundle dir permissions to 0755 so they can be read by in-cluster tooling.
```
Comment (hasbro17) [inline CHANGELOG.md:23]: Again, probably the CHANGELOG fragment itself but we're missing the PR link.
```suggestion
- **Breaking change**: Set bundle dir permissions to 0755 so they can be read by in-cluster tooling. ([#3129](https://github.com/operator-framework/operator-sdk/pull/3129))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,59 @@
...
+- the internal OLM client retrieves existing OLM versions correctly now that the returned list of CSVs is indexed properly. ([#2969](https://github.com/operator-framework/operator-sdk/pull/2969))
+- The Ansible Operator proxy will now return a 500 if it cannot determine whether a resource is virtual or not, instead of continuing on and skipping the cache. This will prevent resources that should have ownerReferences injected from being created without them, which would leave the user in a state that cannot be recovered without manual intervention. ([#3112](https://github.com/operator-framework/operator-sdk/pull/3112))
+- The Ansible Operator proxy no longer will attempt to cache non-status  subresource requests. This will fix the issue where attempting to get Pod logs returns the API Pod resource instead of the log contents. ([#3103](https://github.com/operator-framework/operator-sdk/pull/3103))
+- do not generate a package manifest when 'generate csv --make-manifests=true'. ([#3014](https://github.com/operator-framework/operator-sdk/pull/3014))
```
Comment (hasbro17) [inline CHANGELOG.md:50]: Nit:
```suggestion
- Do not generate a package manifest when 'generate csv --make-manifests=true'. ([#3014](https://github.com/operator-framework/operator-sdk/pull/3014))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,59 @@
...
+- Fixed issue to convert variables with numbers for Ansible based-operator. ([#2842](https://github.com/operator-framework/operator-sdk/pull/2842)). ([#2842](https://github.com/operator-framework/operator-sdk/pull/2842))
+- fix issue faced when the `healthz` endpoint is successfully called. ([#2842](https://github.com/operator-framework/operator-sdk/pull/2842))
```
Comment (hasbro17) [inline CHANGELOG.md:52]:
```suggestion
- Fix issue faced when the `healthz` endpoint is successfully called. ([#2842](https://github.com/operator-framework/operator-sdk/pull/2842))
```

File: CHANGELOG.md (hunk 2 above)
Comment (estroz) [reply CHANGELOG.md:23]: This should've been added automatically. Will investigate.

File: CHANGELOG.md (hunk 3 above)
Comment (estroz) [reply CHANGELOG.md:50]: The changelog generator should impose some formatting rules like this.

File: website/content/en/docs/migration/v0.18.0.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+## Remove the file `/bin/ao-logs` for Ansible based-operators
+
+The `inotify-tools` as a dependency of Ansible based-operator images which was deprecated in the version `v0.16.0` was removed. This dependency was used to output the Ansible logs in a side-car container called `ansible`, which has no longer been scaffolded by default. In this way, if you have any customization using this dependency be aware that it is no longer in the Ansible based-operator image and then, feel free to remove this file `/bin/ao-logs` for your projects since it has no longer usage. For further information also see the section `Remove Ansible container sidecar` on the migration guide of the version `v0.16.0`.
+
+_See [#2852](https://github.com/operator-framework/operator-sdk/pull/2852) for more details._
+
+## Modify permissions on your bundle manifests directory
+
+Run `chmod 0755 deploy/olm-catalog/<operator-name>/manifests` to update your operator's bundle directory permissions.
+
```
Comment (estroz) [inline website/content/en/docs/migration/v0.18.0.md:29]:
```suggestion

_See [#3129](https://github.com/operator-framework/operator-sdk/pull/3129) for more details._
```

File: CHANGELOG.md (hunk 3 above)
Comment (hasbro17) [reply CHANGELOG.md]: Yeah, or better yet we try to catch this when the Fragment PR is added.
So that the release manager doesn't have to fix it all up.

**Pull request merged**